tests/src/model/pages/run-image-page.ts

```diff
@@ -109,4 +109,11 @@ export class RunImagePage extends BasePage {
     await hostPort.fill(customPortMapping.split(':')[0]);
     await containerPort.fill(customPortMapping.split(':')[1]);
   }
+
+  async setHostPortMapping(customPort: string): Promise<void> {
```

Comment:
not necessary anymore: https://github.com/containers/podman-desktop/pull/5304/files#diff-c66b8072e74147f14715eb1f73a95799bc9deb70efd9ead3f3e32d32e4ed4f85R51

Reply:
removed duplicit method

Reply:
removed duplicit method